CauldronMods/Controller/Villains/Tiamat/CharacterCards/TiamatTurnTakerController.cs

```diff
@@ -147,12 +147,10 @@ public void MoveStartingCards()
 
         public override bool IsGameWinnable()
         {
-            
             if(GameController.FindCardController(TurnTaker.FindCard("WinterTiamatCharacter")) is HydraWinterTiamatCharacterCardController)
             {
-                return TurnTaker.GetCardsWhere((Card c) => c.IsCharacter && c.IsInGame).Count() == 6;
+                return TurnTaker.GetCardsWhere((Card c) => c.IsCharacter && c.IsInGame).Count() < 6;
```

Comment:
I changed this to less than since the character card reads "The heroes win if there are 6 decapitated heads." Meaning that if, for whatever reason, there were more than 6 heads in play, the game would still be winnable.